QEfficient/finetune/summarize_infer.py

```diff
@@ -0,0 +1,87 @@
+# -----------------------------------------------------------------------------
```

Comment:
The filename _infer.py doesn't sit well with what we are trying to do here?

Reply:
nit: this goes away, the finetuned checkpoints are directly integrated with QEff. inference stack